src/cli/src/export.rs

```diff
@@ -56,6 +56,11 @@ pub struct ExportCommand {
 
     /// Directory to put the exported data. E.g.: /tmp/greptimedb-export
     /// for local export.
+    ///
+    /// if both `output_dir` and `s3` are set, `output_dir` will be only used for
```

Comment:
Non-orthogonal arguments are bad. How about separate the sql export destination and the data export destination to two different arguments?